src/main/java/org/folio/notes/service/impl/NoteTenantService.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.notes.service.impl;
+
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+import org.folio.notes.service.NoteTypesService;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.folio.spring.service.TenantService;
+
+@Service
+@Primary
+public class NoteTenantService extends TenantService {
+
+  private final NoteTypesService noteTypesService;
+
+  public NoteTenantService(JdbcTemplate jdbcTemplate,
+                           FolioExecutionContext context,
+                           FolioSpringLiquibase folioSpringLiquibase,
+                           NoteTypesService noteTypesService) {
+    super(jdbcTemplate, context, folioSpringLiquibase);
+    this.noteTypesService = noteTypesService;
+  }
+
+  @Override
+  public void loadReferenceData() {
+    super.loadReferenceData();
```

Comment:
I don't think we should call `super.loadReferenceData()`.
It's asking to implement this method
![image](https://user-images.githubusercontent.com/88006020/174027778-84dedddd-83c2-4b00-89e6-ff0cbe502f02.png)